collections.md

```diff
@@ -2402,7 +2402,7 @@ $collection->reduce(function (int $carry, int $item) {
 // 10
 ```
 
-The `reduce` method also passes array keys in associative collections to the given callback:
+The `reduce` method also passes array keys to the given callback:
```

Comment:
The method always receives the key. Having access to the key is also useful for lists.